lib/chef/provider/aws_rds_instance.rb

```diff
@@ -0,0 +1,35 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSProvider
+  REQUIRED_OPTIONS = %w(db_instance_identifier allocated_storage engine
+                        db_instance_class master_username master_user_password)
+
+  OTHER_OPTIONS = %w(engine_version multi_az iops publicly_accessible db_name db_port)
+
+  def update_aws_object(instance)
+    Chef::Log.warn("aws_rds_instance does not support modifying a started instance")
```

Comment:
Cool, I plan on circling back around to adding update support to this once @tylercloke and I are able to stand up all the resources we need.